firebase.json

```diff
@@ -26,7 +26,7 @@
       "host": "127.0.0.1"
     },
     "firestore": {
-      "port": 8080,
+      "port": 8081,
```

Comment:
remove these port changes in the application

Reply:
done